document/core/exec/instructions.rst

```diff
@@ -591,13 +591,15 @@ Memory Instructions
 
 8. Pop the value :math:`\I32.\CONST~n` from the stack.
 
-9. Either, try :ref:`growing <grow-mem>` :math:`\X{mem}` by :math:`n` :ref:`pages <page-size>`:
+9. Let :math:`\X{err}` be the |i32| value :math:`2^{32}-1`, for which :math:`\signed_{32}^{-1}(\X{err})` is :math:`-1`.
```

Comment:
I think you got it backwards. It should be
```
:math:`\signed_{32}(\X{err})` is :math:`-1`
```

Reply:
Oops, right, thanks, fixed.